CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## [In git](https://github.com/strongqa/howitzer/compare/v2.1.1...master)
+## [In git](https://github.com/strongqa/howitzer/compare/v2.1.2...master)
+
+### New Features
+- [#276](https://github.com/strongqa/howitzer/issues/276) Add support Capybara v3
+
+## [v2.1.2](https://github.com/strongqa/howitzer/compare/v2.1.1...v2.1.2)
```

Comment:
What are you doing is suitable for releasing, but it should be with separate commit with tagging. Please just add 1 line with feature to old file.